hybrasyl/Game.cs

```diff
@@ -199,6 +202,14 @@ public static void Main(string[] args)
             LogLevel = Hybrasyl.Constants.DEFAULT_LOG_LEVEL;
             Assemblyinfo = new AssemblyInfo(Assembly.GetEntryAssembly());
 
+            //set up service endpoint for ControlService
+
+            var host = new WebServiceHost(typeof(ControlService), new Uri("http://localhost:4949/ControlService"));

```

Comment:
OK here, I'll take on the task of extending the XML for config to handle configuring this, along with the "shutdown password" / token.

We should also use a more standard `X-Access-Token` versus exposing the key in the GET request.